workflow-templates/validate-tsoa.yml

```diff
@@ -0,0 +1,19 @@
+name: Validate TSOA
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: "Validate TSOA"
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
```
    if: github.repository != 'meetlara/clerk-service'
```

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution

Reply:
```
jobs:
  build:
    name: "Validations (no solo tsoa)"
    runs-on: ubuntu-latest
    env:
      EXCLUDED_REPOS: "meetlara/airflow meetlara/clerk-service" 
    steps:
      - name: run tsoa
        ...
         if: ${{ !contains(env.EXCLUDED_REPOS, github.repository) }}

Reply:
perfectou